build/makelib/helm.mk

```diff
@@ -38,6 +38,8 @@ $(HELM):
 define helm.chart
 $(HELM_OUTPUT_DIR)/$(1)-$(VERSION).tgz: $(HELM) $(HELM_OUTPUT_DIR) $(shell find $(HELM_CHARTS_DIR)/$(1) -type f)
 	@echo === helm package $(1)
+	@$(HELM) repo add ceph-csi-operator https://ceph.github.io/ceph-csi-operator --force-update
+	@cd $(HELM_CHARTS_DIR)/$(1) && $(HELM) dependency build
```

Comment:
Is this the same as the `helm.dependency.build` target? Can we instead call that target?

Reply:
done